src/Rings/mpoly-ideals.jl

```diff
@@ -1968,6 +1998,11 @@ false
   return isempty(lg) || all(iszero, lg)
```

Comment:
All elements of the empty list are zero
```suggestion
  return all(iszero, lg)
```